src/applications/vaos/tests/new-appointment/components/DateTimeRequestPage.unit.spec.jsx

```diff
@@ -10,6 +11,12 @@ import { waitFor } from '@testing-library/dom';
 import { Route } from 'react-router-dom';
 
 describe('VAOS <DateTimeRequestPage>', () => {
+  beforeEach(() => {
```

Comment:
The calendar pages have different logic and different days available depending on min date and day of the week logic.